dwave/system/samplers/leap_hybrid_sampler.py

```diff
@@ -126,11 +132,12 @@ def properties(self):
 
     @property
     def parameters(self):
-        """dict[str, list]: solver parameters in the form of a dict, where keys are
+        """dict[str, list]: Solver parameters in the form of a dict, where keys are
         keyword parameters accepted by a SAPI query and values are lists of properties in
-        :attr:`.LeapHybridSampler.properties` for each key.
+        :attr:`~dwave.system.samplers.LeapHybridSampler.properties` for each key.
```

Comment:
should be ``` :attr:`.properties` ``` so as to not break doc inheritance

Reply:
See above. Sphinx unfortunately incorrectly links `:attr:`.properties`` to the properties of a different class, which will frustrate/confuse users.